### DISCUSSION TITLE
New Pattern - Azure Machine Learning / Azure Synapse Analytics - Classical Machine Learning

### DISCUSSION THREAD

**Discussion by ShaneOsborne** [Azure/mlops-v2#11] [Ideas]
Good morning

I would love to own/create a pattern based on the current work for using 'Azure Synapse Analytics - Classical Machine Learning' 

If this is of interested, please let me know and will work on it.

Thanks

Shane

**Comment by setuc** [comment]
Yes, this would be a good contribution. Let us know what ideas do you have in mind.
